docs/notebooks/notdiamond_custom_routing.ipynb

```diff
@@ -0,0 +1,476 @@
+{
```

Comment:
It is up to you as an author if you would like to include notebook outputs (sometimes they are useful), but also can be distracting. Consider clearing outputs